test/CMakeLists.txt

```diff
@@ -200,6 +200,10 @@ set(XSIMD_TESTS
     test_utils.hpp
 )
 
+if(NOT MSVC)
```

Comment:
Wouldn't that be more "error-proof" to do that check in `xsimd_scalar.hpp`, as you did for `__minGW32__`?

Reply:
I don't think `_GNU_SOURCE` will ever (and shouldn't) be defined with NSVC, and if so then an issue should be fixed.

Reply:
ack